src/core/utils/index.js

```diff
@@ -336,3 +348,39 @@ export const getAddonTypeFilter = (addonType, _config = config) => {
 
   return ADDON_TYPE_THEMES_FILTER;
 };
+
+export const getExtensionTypeAction = (type = ADDON_TYPE_EXTENSION) => {
+  let action = TRACKING_TYPE_EXTENSION;
+
+  if (type === ADDON_TYPE_STATIC_THEME) {
+    action = TRACKING_TYPE_STATIC_THEME;
+  }
+
+  return action;
+};
+
+export const getAddonCategory = ({
+  type = ADDON_TYPE_EXTENSION,
+  installCategory = INSTALL_CATEGORY,
```

Comment:
I think this would be clearer without any default values. You don't seem to ever call it without a `type` currently, and you do call it twice without an `installCategory`, but I think the calling code would be clearer if you always passed both.